index.html

```diff
@@ -42,7 +42,7 @@
         <a href="https://faculty.sites.uci.edu/davis/" target="_blank">
           <img
             class="logo"
-            src="images/Coastal_Dynamics_Sketches_Background.png"
+            src="/images/Coastal_Dynamics_Sketches_Background.png"
```

Comment:
Not sure why is this slash automatically added, but this is causing a logo loading issue when it is hosted. So we might need to change it back.

Reply:
It's automatically added when I saved the file